model/service_logs/v1/cluster_log_type.model

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+class ClusterLog {
+    Severity        String
```

Comment:
What is the meaning of this? And what are the possible values? Consider using an enumerated type:

```
// Indicates the severity of a log entry.
enum Severity {
        // Indicates that the log entry is of low severity.
        Low

        // Indicates that the log entry is of high severity.
        High
}
```
Then the attribute should be declared like this:

```
// Indicates the severity of the log entry.
Severity Severity
```

Reply:
This is the severity of the log - we support: debug,info,warning,error,fatal.
Sure, I can add this as enum